Assets/MixedRealityToolkit/_Core/Definitions/Devices/DeviceInputType.cs

```diff
@@ -60,11 +64,13 @@ public enum DeviceInputType
         Start,
         Menu,
         Hand,
+        ThumbPress,
+        ThumbTouch,
+        ThumbNearTouch,
```

Comment:
What is "Near Touch" for a thumbstick?

Reply:
I reverted this enum in a PR against his branch.. Nothing to see here.